src/WarcraftLegacies.Shared/Constants.cs

```diff
@@ -1,3 +1,4 @@
+﻿#pragma warning disable
```

Comment:
This will get removed when Constants.cs and Regions.cs are regenerated, which happens on every W3X-->MapData process. I think we'll need to prepend it on generation somehow?

Reply:
Cool, so sounds like we should
1. Prepend the pragma warning disable after generation, so this can be merged
2. Ask Arxos to make the same change upstream